test/none.js

```diff
@@ -0,0 +1,43 @@
+var assert = require('assert');
+
+var R = require('..');
+
+
+describe('none', function() {
+    var even = function(n) {return n % 2 === 0;};
+    var T = function() {return true;};
+
+    it('returns true if no elements satisfy the predicate', function() {
+        assert.strictEqual(R.none(even, [1, 3, 5, 7, 9, 11]), true);
+    });
+
+    it('returns false if any element satisfies the predicate', function() {
+        assert.strictEqual(R.none(even, [1, 3, 5, 7, 8, 11]), false);
+    });
+
+    it('returns true for an empty list', function() {
+        assert.strictEqual(R.none(T, []), true);
+    });
+
+    it('short-circuits on first false value', function() {
```

Comment:
Yes, we should remove it from `R.all` as well.

It seemed to make sense at the time, but I wouldn't want to imagine how we'd try to explain it if we were ever to re-implement in a concurrent alternative on some imaginary multi-threaded version of the language.  So it's definitely in fact an implementation detail.